test/tap/layouts-test.js

```diff
@@ -375,6 +375,122 @@ test('log4js layouts', (batch) => {
       assert.end();
     });
 
+    t.test('%m should apply util.format on data', (assert) => {
+      const eventWithSeveralDataEntry = JSON.parse(JSON.stringify(event));
+      eventWithSeveralDataEntry.data = [
+        'This %s a %s like other ones',
+        "isn't",
+        'test',
+      ];
+      testPattern(
+        assert,
+        layout,
+        eventWithSeveralDataEntry,
+        tokens,
+        '%m',
+        "This isn't a test like other ones"
+      );
+      assert.end();
+    });
+
+    t.test(
+      '%m{P}, with P been an integer, shoud only consider data.slice( P )',
+      (assert) => {
+        const eventWithSeveralDataEntry = JSON.parse(JSON.stringify(event));
+        eventWithSeveralDataEntry.data = [
+          'This %s a %s like other ones',
+          "isn't",
+          'test',
+        ];
+        testPattern(
+          assert,
+          layout,
+          eventWithSeveralDataEntry,
+          tokens,
+          '%m{1}',
+          "isn't test"
+        );
+        assert.end();
+      }
+    );
+
+    t.test('%m{0, 1} should behave like a dummy layout', (assert) => {
+      const eventWithSeveralDataEntry = JSON.parse(JSON.stringify(event));
+      eventWithSeveralDataEntry.data = [
+        'This %s a %s like other ones',
+        "isn't",
+        'test',
+      ];
+      testPattern(
+        assert,
+        layout,
+        eventWithSeveralDataEntry,
+        tokens,
+        '%m{0, 1}',
+        'This %s a %s like other ones'
+      );
+      assert.end();
+    });
+
+    t.test('%m{1, 2} shoud only consider data.slice( 1, 2 )', (assert) => {
+      const eventWithSeveralDataEntry = JSON.parse(JSON.stringify(event));
+      eventWithSeveralDataEntry.data = [
+        'This %s a %s like other ones',
+        "isn't",
+        'test',
+      ];
+      testPattern(
+        assert,
+        layout,
+        eventWithSeveralDataEntry,
+        tokens,
+        '%m{1,2}',
+        "isn't"
+      );
+      assert.end();
+    });
+
+    t.test('%m{1, 2} shoud only consider data.slice( 1, 2 )', (assert) => {
```

Comment:
For consistency and typo.
```suggestion
    t.test('%m{1,2} should only consider data.slice(1, 2)', (assert) => {
```